doc/configuration.rst

```diff
@@ -4,7 +4,7 @@ Configuration
 All configuration can be done by adding a configuration files. They are looked for in:
 
  * ``/etc/luigi/client.cfg``
```

Comment:
I don't completely recall, but I guess that my main issue was that it is unobvious what `$CWD/client.cfg` is about, while `/etc/luigi/client.cfg` is more self-explanatory , so I focussed on the former one and left the latter as-is